packages/connection-form/src/components/connection-form.spec.tsx

```diff
@@ -28,23 +32,30 @@ const noop = (): any => {
 const saveAndConnectText = 'Save & Connect';
 const favoriteText = 'FAVORITE';
 
+let preferences: PreferencesAccess;
```

Comment:
Very much an optional suggestion, but I'd always put these test-local variables into the `describe()`/`context()`/etc. block where the initializing `before()`/`beforeEach()` call also is, that makes it a bit harder to accidentally misuse